profile/README.md

```diff
@@ -33,3 +28,17 @@ If you are interested in what we do as engineers or data scientists, check out o
 * [Machine Learning atAlan](https://medium.com/alan/machine-learning-atalan-e436e0ae721d?source=rss----b2cb698c4e73---4)
 * [Compose with temporal logic](https://medium.com/alan/compose-with-temporal-logic-e0b77b860b32?source=rss----b2cb698c4e73---4)
 <!--END_SECTION:feed-->
+
+</td>
+<td  width="50%" valign="baseline">
+      
+#### Sponsored by us? 💚
+
+<!-- todo: add sponsorship program link -->
+We regularly sponsor some open-source projects on GitHub as part of our Sponsorship Program.
+  
+You can get in touch and/or send feedback [here](https://forms.gle/YxxyJadt31w9RhXB6)!
+  
+  </td>
+  </tr>
```

Comment:
unfortunately this doesn't work because markdown recognises 4-spaces indented code as code blocks

    like this

😢 

![image](https://user-images.githubusercontent.com/24496417/175893497-3487a298-a88b-45c5-97eb-ae6929e73acf.png)